### PR TITLE
Implement start component

### PR DESCRIPTION
**Pull request by byeolbit**
Implement start component which contains features below:
- Button for load from local repository
- Button for clone from GitHub account

ISSUE = #4

### CODE REVIEW

File: src/app/start/start.component.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.sign-in span {
+  width: 146px;
+  height: 14px;
+  font-size: 14px;
+  color: #ffffff;
+  line-height: 18px;
+  text-align: center;
+  font-weight: lighter;
+  vertical-align: middle;
+}
```
Comment (byeolbit) [inline src/app/start/start.component.scss]: CSS style selectors name have to be changed.
I'll solve this later.

**Comment by byeolbit** [comment]
@haeungun PTAL :)

File: src/app/start/start.component.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    });
+  }
+
+  private navigateToEditor(result) {
+    if (result.success) {
+      this.router.navigate(['/editor',{'src' : result.path }]);
+      this.jekyllService.passJekyllProcess(result.child);
+    }
+  }
+
```
Comment (byeolbit) [inline src/app/start/start.component.ts]: This series of code(line 51~78) is duplicated code(It is in `repository.component.ts`).
This seems not beautiful.
I think we have to change it later.
I'll add this problem as new issue.

File: src/app/start/start.component.scss (hunk 1 above)
Comment (haeungun) [reply src/app/start/start.component.scss]: Okay. How do you think of leaving this as an issue?

File: src/app/start/start.component.ts (hunk 2 above)
Comment (haeungun) [reply src/app/start/start.component.ts]: Great. I think it would be nice to refactor later.

**Comment by haeungun** [comment]
lgtm

**Pull request merged**